starlette/responses.py

```
@@ -341,7 +341,7 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
                 stat_result = await anyio.to_thread.run_sync(os.stat, self.path)
                 self.set_stat_headers(stat_result)
             except FileNotFoundError:
-                raise RuntimeError(f"File at path {self.path} does not exist.")
+                pass
             else:
                 mode = stat_result.st_mode
```

2 lines below, we have another `RuntimeError`.

Also, this is a breaking change.

---

We can raise a new exception derived from both `RuntimeError` and `FileNotFoundError`. This way we can keep backward compatibility.
```python
class NotFoundError(FileNotFoundError, RuntimeError): ...
```